### ISSUE TITLE
TypeError: subcon should be a Construct field

### ISSUE THREAD

**Issue by danielfaust** [rytilahti/python-eq3bt#13]
The following exception is raised on a Raspberry Pi 3 during the initial import of the library.

```
Traceback (most recent call last):
  File "bt-window.py", line 2, in <module>
    from eq3bt import Thermostat
  File "/usr/local/lib/python2.7/dist-packages/eq3bt/__init__.py", line 2, in <module>
    from .eq3btsmart import Thermostat, TemperatureException, Mode
  File "/usr/local/lib/python2.7/dist-packages/eq3bt/eq3btsmart.py", line 17, in <module>
    from .structures import *
  File "/usr/local/lib/python2.7/dist-packages/eq3bt/structures.py", line 79, in <module>
    "cmd" / Const(Int8ub, PROP_INFO_RETURN),
  File "/usr/local/lib/python2.7/dist-packages/construct/core.py", line 1277, in __init__
    super(Const, self).__init__(subcon)
  File "/usr/local/lib/python2.7/dist-packages/construct/core.py", line 287, in __init__
    raise TypeError("subcon should be a Construct field")
TypeError: subcon should be a Construct field
```

**Comment by tmad** [comment]
You need to downgrade construct to 2.8.21.

PR: https://github.com/rytilahti/python-eq3bt/pull/14

**Comment by rytilahti** [comment]
This was caused by a change in the construct lib and needs to be fixed in order to support the newer versions, see https://github.com/construct/construct/commit/561fede583f2e96ca6a614c6291a0b887539bc8d . It's a mess, as as soon as this gets changed older versions of construct won't work anymore either, but this has to be fixed here so I'm reopening this.

**Comment by rytilahti** [comment]
Fixed with 2ebabef9e0e5dd6020c418adce178eec292c9fc0 - 0.1.8 is now available from pypi.
